custom/covid/management/commands/update_cases.py

```diff
@@ -45,6 +60,8 @@ def handle(self, domain, case_type, **options):
         else:
             user_id = SYSTEM_USER_ID
 
+        self.output_file = options["output_file"]
```

Comment:
`options.pop("output_file", None)` would keep this from getting added to `extra_options`